src/core-services/account/schemas/account.graphql

```diff
@@ -136,6 +136,19 @@ input RemoveAccountEmailRecordInput {
   email: Email!
 }
 
+"Defines groups to add given user / account to thereby giving that user certain permissions"
+input SetUserPermissionsInput {
+
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+
+  "the groups to add a user / account to"
+  groups: [String!]!
```

Comment:
I just want to clarify something about this syntax in case it doesn't do exactly what you expect.

This means:
- groups must be an array
- groups must be present
- groups must only contain strings, no nulls

This does _not_ mean:
- at least one group is required

That is, `groups: []` would be acceptable. If you were trying to require at least one element in the groups array that is outside the GraphQL scope and needs to be enforced by the resolver itself.

Reply:
@rosshadden ... thanks for the explanation ... I've updated the mutation function validation to check that there is at least one group provided in this mutation